spec/queries/control_rate_query_v2_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe ControlRateQueryV2 do
+  let(:organization) { create(:organization, name: "org-1") }
+  let(:user) { create(:admin, :manager, :with_access, resource: organization, organization: organization) }
+  let(:facility_group_1) { FactoryBot.create(:facility_group, name: "facility_group_1", organization: organization) }
+  let(:query) { ControlRateQueryV2.new }
+
+  let(:june_1_2018) { Time.parse("June 1, 2018 00:00:00+00:00") }
+  let(:june_1_2020) { Time.parse("June 1, 2020 00:00:00+00:00") }
+  let(:june_30_2020) { Time.parse("June 30, 2020 00:00:00+00:00") }
+  let(:july_2020) { Time.parse("July 15, 2020 00:00:00+00:00") }
+  let(:jan_2019) { Time.parse("January 1st, 2019 00:00:00+00:00") }
+  let(:jan_2020) { Time.parse("January 1st, 2020 00:00:00+00:00") }
+  let(:july_2018) { Time.parse("July 1st, 2018 00:00:00+00:00") }
+  let(:july_1_2020) { Time.parse("July 1st, 2020 00:00:00+00:00") }
```

Comment:
This is the sort of use case that the `june_2021` helper was created for. I thought we could use a common set of known times and boundaries that we're interested in, in all our reporting tests as opposed to creating new times and boundaries in each test.

Reply:
I like the idea of having some common date helpers, but so far I prefer the readability of having the actual time stamps created in the test file itself.  Its simple and explicit and clear, and I don't have to do relative date math in my head.

Also, one thing we've ran into in the past is figuring out what timezone `now - 2.years` is operating in -- with a iso8601(ish) timestamp string like above, there is no confusion.  Its right there in the string that gets parsed.